[2022-08-05 13:31:34,183] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: a_test.print_date scheduled__2022-08-04T04:00:12.650445+00:00 [queued]>
[2022-08-05 13:31:34,192] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: a_test.print_date scheduled__2022-08-04T04:00:12.650445+00:00 [queued]>
[2022-08-05 13:31:34,193] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-05 13:31:34,193] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-05 13:31:34,193] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-05 13:31:34,209] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): print_date> on 2022-08-04 04:00:12.650445+00:00
[2022-08-05 13:31:34,214] {standard_task_runner.py:52} INFO - Started process 67212 to run task
[2022-08-05 13:31:34,219] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'a_test', 'print_date', 'scheduled__2022-08-04T04:00:12.650445+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/var/folders/0s/6831mwss3ngbr2rh0337_18r0000gn/T/tmpr9cbtdph', '--error-file', '/var/folders/0s/6831mwss3ngbr2rh0337_18r0000gn/T/tmpy1c_mdt3']
[2022-08-05 13:31:34,221] {standard_task_runner.py:80} INFO - Job 55: Subtask print_date
[2022-08-05 13:31:39,277] {task_command.py:371} INFO - Running <TaskInstance: a_test.print_date scheduled__2022-08-04T04:00:12.650445+00:00 [running]> on host Fu-Changs-MacBook-Pro.local
[2022-08-05 13:31:44,349] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=anappleaday1984@gmail.com
AIRFLOW_CTX_DAG_OWNER=fu
AIRFLOW_CTX_DAG_ID=a_test
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2022-08-04T04:00:12.650445+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-04T04:00:12.650445+00:00
[2022-08-05 13:31:44,351] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/0s/6831mwss3ngbr2rh0337_18r0000gn/T
[2022-08-05 13:31:44,351] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'date']
[2022-08-05 13:31:44,368] {subprocess.py:85} INFO - Output:
[2022-08-05 13:31:44,416] {subprocess.py:92} INFO - Fri Aug  5 13:31:44 CST 2022
[2022-08-05 13:31:44,416] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-05 13:31:44,440] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=a_test, task_id=print_date, execution_date=20220804T040012, start_date=20220805T053134, end_date=20220805T053144
[2022-08-05 13:31:44,463] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-05 13:31:44,488] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
